### PR TITLE
Summary-screen changes and localization

### PR DESCRIPTION
**Pull request by abishekTa-egov**
Made summary screen datamgmt , and css changes, localiztions in user-access-roles

## Release Notes

- **New Features**
	- Enhanced styling for health-related components with updated CSS.
	- Improved user interface for the Assumptions List and Campaign Boundary components, including new buttons and layout adjustments.
	- Added new customization options for the search results configuration.
	- Introduced dynamic state management and improved rendering logic in various components.
	- New methods added to the UICustomizations for enhanced functionality.

- **Bug Fixes**
	- Resolved issues with the popup headings in the User Access component for better clarity.

- **Style**
	- Various components received style updates for improved visual consistency and responsiveness, including background color changes and new class additions.

- **Chores**
	- Cleaned up unused imports and optimized the code structure across multiple components.

### CODE REVIEW

**Review by nabeelmd-eGov — APPROVED** [review]

File: health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js [hunk 1]
```diff
@@ -143,7 +143,51 @@ export const UICustomizations = {
...
       }
-    },
+      if(key==="MICROPLAN_STATUS"){
+        if(value &&value!="NA"){
+        return t(value)
+        }
+      else {
+        return (
+          <div>
+            <p>NA</p>
```
Comment (nipunarora-eGov) [inline health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js:153]: Make sure all strings are localised

File: health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/SummaryScreen.js [hunk 2]
```diff
@@ -316,7 +319,7 @@ const SummaryScreen = ({ props: customProps }) => {
                     code: "USER_ACCESS_MGMT",
                 },
             ],
-            activeByDefault: "card7",
+            activeByDefault: "card2",
```
Comment (nipunarora-eGov) [inline health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/SummaryScreen.js:322]: activeByDefault should be first card

**Review by nabeelmd-eGov — DISMISSED** [review]

File: health/micro-ui/web/micro-ui-internals/example/public/index.html [hunk 3]
```diff
@@ -16,7 +16,7 @@
   /> -->
   <link rel="stylesheet" href="https://unpkg.com/@egovernments/digit-ui-css@1.8.2-beta.34/dist/index.css" />
   <link rel="stylesheet" href="https://unpkg.com/@egovernments/digit-ui-components-css@0.0.2-beta.39/dist/index.css" />
-  <link rel="stylesheet" href="https://unpkg.com/@egovernments/digit-ui-health-css@0.1.9/dist/index.css" />
+  <link rel="stylesheet" href="https://unpkg.com/@egovernments/digit-ui-health-css@0.2.0/dist/index.css" />
```
Comment (nipunarora-eGov) [inline health/micro-ui/web/micro-ui-internals/example/public/index.html:19]: keep it 0.1.10 @abishekTa-egov everywhere

File: health/micro-ui/web/micro-ui-internals/packages/css/src/components/microplan.scss [hunk 4]
```diff
@@ -128,42 +128,78 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 1rem;
+}
+
+.edit-button {
+  background-color: transparent;
+  border: none;
+  color: #f58220; /* Orange color for the Edit button */
```
Comment (nabeelmd-eGov) [inline health/micro-ui/web/micro-ui-internals/packages/css/src/components/microplan.scss:141]: wrong color code

File: health/micro-ui/web/micro-ui-internals/packages/css/src/components/microplan.scss [hunk 5]
```diff
@@ -128,42 +128,78 @@
...
-  border-spacing: 0 1rem;     /* 1rem space between rows */
+  border-collapse: collapse;
+  table-layout: fixed;
 }
 
-.as-table-row{
+.as-table-row {
   display: table-row;
-  
+  border-bottom: 1px solid #161515;
```
Comment (nabeelmd-eGov) [inline health/micro-ui/web/micro-ui-internals/packages/css/src/components/microplan.scss:154]: wrong color code

File: health/micro-ui/web/micro-ui-internals/packages/css/src/components/microplan.scss [hunk 6]
```diff
@@ -128,42 +128,78 @@
...
+  margin-bottom: 1rem;
+}
+
+.as-label-field { 
+  display: grid;               /* Enable grid layout */
+  grid-template-columns: 1fr 2fr; /* Define two columns with different widths */
+  gap: 0rem;                  /* Space between items (adjust as needed) */
+  overflow: auto;             /* Allow overflow if content is larger */
+  align-items: center;
+  border-bottom: '1px solid #e0e0e0'        /* Align items vertically in the center */
```
Comment (nabeelmd-eGov) [inline health/micro-ui/web/micro-ui-internals/packages/css/src/components/microplan.scss:204]: wrong color code

**Review by nabeelmd-eGov — APPROVED** [review]

File: health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js [hunk 7]
```diff
@@ -138,12 +138,56 @@ export const UICustomizations = {
...
+        );
+      }}
+      if(key==="CAMPAIGN_TYPE"){
+        if(value &&value!="NA"){
+        return t(value)
+        }else {
+        return (
+          <div>
+            <p>{t("NA")}</p>
+          </div>
```
Comment (nipunarora-eGov) [inline health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js:175]: make the same changes in example/src/UICustomizatiosn

File: health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/CampaignBoundary.js [hunk 8]
```diff
@@ -294,60 +294,65 @@ const CampaignBoundary = ({ customProps }) => {
...
+                                    className="view-more"
+                                >
+                                    View More
+                                </div>
+                            ) : (!boundaryStatus?.[ind + 2]) ? (
+                                <div
+                                    onClick={() => handleViewMore(ind + 2)}
+                                    className="view-more"
+                                >
+                                    View Less
```
Comment (nipunarora-eGov) [inline health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/CampaignBoundary.js:349]: Locailisations here as well

**Review by nipunarora-eGov — DISMISSED** [review]

**Review by nipunarora-eGov — APPROVED** [review]

**Pull request merged**